appserver/admingui/core/src/main/resources/login.jsf

```diff
@@ -60,6 +60,12 @@
         setPageSessionAttribute(key="byPassLogin" value="true");
     }
     setSessionAttribute(key="showLogoutButton" value="#{! pageSession.byPassLogin}")
+    
+    gf.restRequest(endpoint="#{request.scheme}://#{sessionScope.serverName}:#{sessionScope.serverPort}/management/domain/asadmin-recorder-enabled"
```

Comment:
Upon review, adding stuff to here is probably a little messy.
I shall look into initialising the session attributes with the other session attributes in the `GuiUtil` class